config/initializers/govuk_components.rb

```diff
@@ -0,0 +1,5 @@
+Govuk::Components.configure do |config|
+  # config.brand = 'dfe'
+  config.default_header_service_name = 'Early years child development training'
```

Comment:
We already have this is `config.service_name`

Reply:
i don't think the configuration is needed any longer, so deleted

Reply:
Actually, turns out is is needed for the header component - previously it was being passed in.